receiver/hostmetricsreceiver/internal/scraper/processesscraper/processes_scraper.go

```diff
@@ -50,7 +87,54 @@ func (s *scraper) start(context.Context, component.Host) error {
 }
 
 func (s *scraper) scrape(_ context.Context) (pdata.MetricSlice, error) {
+	now := pdata.TimestampFromTime(time.Now())
+
 	metrics := pdata.NewMetricSlice()
-	err := appendSystemSpecificProcessesMetrics(metrics, s.startTime, s.misc)
+	metrics.EnsureCapacity(metricsLength)
+
+	processMetadata, err := s.getProcessesMetadata()
+	if err != nil {
+		err = scrapererror.NewPartialScrapeError(err, metricsLength)
```

Comment:
Should we just return here?  My reading of getProcessMetadata is that it returns either an error, or a non-empty processMetadata, but not both.

Reply:
Addressed as part of #5536

Reply:
Returning here.